### DISCUSSION TITLE
Potential loss of keyboard focus

### DISCUSSION THREAD

**Discussion by JoelHaslam** [grafana/grafana#39859] [General]
**Steps to reproduce**:
Open up the 'dashboard settings' page from the UI (e.g [here](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editview=settings)) and start pressing tab to cycle through all of the options in the page. Once you reach the end, continue to press tab and try to keep track of your keyboard focus.

**Actual Result**:
The keyboard focus begins to tab through elements that are on the original dashboard UI (and not the settings page that you are currently looking at).

I believe that this is because the settings page is just layered on top of the dashboard, and not it's own individual page. You are able to debug this as a link will appear in the bottom left hand corner to help you understand where the focus currently is, however this can still be confusing and can require excessive tabbing to return to where you want to be.

**Expected Result**:
Keyboard focus to remain within the page that you are looking at and cycle repeatedly cycle through these elements

**Relevant WCAG Criteria:** [2.4.1 bypass-blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks)

**Environment**:
- Grafana version: 8.2.0-beta1
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by JoelHaslam** [comment]
I'm hoping to revive this discussion! This issue is still impacting [play.grafana.org ](https://play.grafana.org/), and I would like to get clarity on whether other users think that this is an impactful issue.

While you are still able to locate the keyboard focus, the loss of it from the page that you are on is a huge inconvenience and as mentioned above, requires excessive 'tabbing' to get this back to being useful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
